articles/quickstart/backend/_includes/_api_create_new_2.md

```diff
@@ -0,0 +1,5 @@
+## Create a Resource Server (API)
+
+In the [APIs](${manage_url}/#/apis) section of the Auth0 dashboard, click **Create API**. Provide a name and an identifier for your API. You will use the identifier as an `audience` later, when you are configuring the access token verification. For **Signing Algorithm**, select **RS256**.
+
```

Comment:
'Provide a name and an identifier for your API, for example "https://quickstarts/api". '

that way you can copy&paste it ;)